### PR TITLE
A simple plugin system for Airflow

### PR DESCRIPTION
**Pull request by mistercrunch**
Check out the docs entry for details

### CODE REVIEW

File: airflow/__init__.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
-__version__ = "1.0.1"
-
 """
 Authentication is implemented using flask_login and different environments can
 implement their own login mechanisms by providing an `airflow_login` module
 in their PYTHONPATH. airflow_login should be based off the
 `airflow.www.login`
 """
+__version__ = "1.0.1"
```
Comment (artwr) [inline airflow/__init__.py]: Maybe minor bump in the version?

**Comment by artwr** [comment]
Sweet, sweet plugins!

File: airflow/__init__.py (hunk 1 above)
Comment (mistercrunch) [inline airflow/__init__.py]: I bump as part of the process when I decide to upload to pypi

**Comment by Yongyiw** [comment]
Is it possible to put different hooks/operators/macros in different directories instead of in one file?

**Comment by mistercrunch** [comment]
@Yongyiw , yes, totally. Within your main plugin file you can import other python modules and associate their content to your `AirflowPlugin`

**Comment by Yongyiw** [comment]
@mistercrunch It works for me. Thanks a lot! 

For future convenience, what i have done is 

1) Created empty files named `__init__.py` in `plugins` directory and its sub-directories eg. `hooks`, `operators`, etc.... 

2) In `AirflowPlugin` type: 

```
     from hooks.your_customized_hook import YourCustomizedHook
     from operators.your_customized_operator import YourCustomizedOperator
```

Details about [python-how-to-import-the-class-within-the-same-directory-or-sub-directory](http://stackoverflow.com/questions/4142151/python-how-to-import-the-class-within-the-same-directory-or-sub-directory)

**Pull request merged**